demo/src/Main.gd

```diff
@@ -62,14 +62,14 @@ func changeTest() -> void:
 		_cur_test_scene.queue_free()
 	
 	_cur_test_scene_index = wrapi(_cur_test_scene_index + 1, 0, test_scenes.size())
-	var instance = test_scenes[_cur_test_scene_index].instance()
+	var instance = test_scenes[_cur_test_scene_index].instantiate()
 	add_child(instance)
 	
 	#for some reason I need to do that in Godot v3.3.1...
 	#the camera in the scene is already setup with current = true and zoom = Vector2(2,2)
 	#but it does not work without setting it here again
-	instance.get_node("Camera2D").current = true
-	instance.get_node("Camera2D").zoom = Vector2(2,2)
+	instance.get_node("Camera2D").make_current()
+	instance.get_node("Camera2D").zoom = Vector2(0.25,0.25)
```

Comment:
camera zoom should be (0.5, 0.5) to work properly? At least for me 0.25 is too much zoomed out.

<img width="691" alt="Screenshot 2024-06-21 at 08 35 52" src="https://github.com/SoloByte/godot-polygon2d-fracture/assets/34277803/0252f666-7b3d-435c-aad9-dadf73f437cb">

Reply:
See https://github.com/SoloByte/godot-polygon2d-fracture/pull/6#issuecomment-2182147499